File: coding-ig/tahun-kabisat/kabisat.py
n = int(input("Masukkan tahun = "))
if n%400 == 0:
    print(str(n)+" adalah tahun Kabisat")
elif n%100 == 0:
    print(str(n)+" bukan tahun Kabisat")
elif n%4 == 0:
    print(str(n)+" adalah tahun Kabisat")
else:
    print(str(n)+" bukan tahun Kabisat")

File: coding-ig/tahun-kabisat/kabisat.c
#include<stdio.h>
int main(){
  int n;
  printf("Masukkan tahun = ");
  scanf("%d",&n);
  if (n%400 == 0){
      printf("%d adalah tahun Kabisat",n);
  }else if (n%100 == 0){
      printf("%d bukan tahun Kabisat",n);
  }else if (n%4 == 0){
      printf("%d adalah tahun Kabisat",n);
  }else{
    printf("%d bukan tahun Kabisat",n);
  }
}
